frb_example/pure_dart_multi/rust/build.rs

```diff
@@ -15,12 +15,24 @@ const RUST_OUTPUT_2: &str = "src/generated_api_2.rs";
 const CLASS_NAME_1: &str = "ApiClass1";
 const CLASS_NAME_2: &str = "ApiClass2";
 
+fn use_c_output_flag() -> bool {
+    #[cfg(feature = "c-output")]
+    return true;
+    false
+}
+
+fn use_extra_c_output_path_flag() -> bool {
+    #[cfg(feature = "extra-c-output-path")]
+    return true;
+    false
+}
```

Comment:
https://doc.rust-lang.org/rust-by-example/attribute/cfg.html

```
cfg!(target_os = "linux") 
```

is a bool

Reply:
I don't get it.

Reply:
Fixed in [cd09bcb](https://github.com/fzyzcjy/flutter_rust_bridge/pull/937/commits/cd09bcb737e03e8875b477db2a0a474837bc7c7e).